src/sdi-refresh-monitor.c

```diff
@@ -512,10 +540,11 @@ static void manage_refresh_inhibit(SnapdClient *source, GAsyncResult *res,
     return;
   // Check if there's at least one snap not marked as "ignore"
   gboolean show_grouped_notification = FALSE;
-  g_autoptr(GSList) snap_list = NULL;
+  g_autoptr(GListStore) snap_list = g_list_store_new(SNAPD_TYPE_SNAP);
   for (guint i = 0; i < snaps->len; i++) {
     SnapdSnap *snap = snaps->pdata[i];
     const gchar *name = snapd_snap_get_name(snap);
+    g_debug("Received notification for inhibited snap %s", name);
```

Comment:
Is this suppose to remain? (I think so, just double checking)

Reply:
Yes. I want to add some extra debug messages, and maybe enable them when launched as a service to simplify fixing bugs (that way we can ask for a journalctl dump).